### DISCUSSION TITLE
Helm installation failed | k8s beginner

### DISCUSSION THREAD

**Discussion by Christopher-06** [milvus-io/milvus#19424] [Q&A and General discussion] [answered]
Hi guys,

I would like to set up a Milvus cluster on 3 servers. With the help of Docker Swarm, such a cluster is already running, but problems like "InvalidMetaCache" or "context deadline exceeded" occur frequently....
So I want to rebuild a cluster using k8s. I am a beginner with kubernetes but have already installed the necessary software on all servers, but I am not getting anywhere. Whenever I try to install milvus with helm, this error appears:

`$ helm install my-release milvus/milvus`
```
Error: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "my-release-pulsar-bookie" namespace: "default" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "my-release-pulsar-broker" namespace: "default" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "my-release-pulsar-proxy" namespace: "default" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first, resource mapping not found for name: "my-release-pulsar-zookeeper" namespace: "default" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first]
```

Also when trying to install milvus with helm on my local PC with minikube, the same error appears. Because, as I said, I'm new to the subject of k8s, I don't know if it's my error or Milvus' error. 

Would be very dear if someone could help me!

Further information:
- minikube version: v1.27.0
- kubectl version: v1.25.0
- helm version: v3.10.0

**Comment by LoveEachDay** [comment, marked as answer]
@Christopher-06 According to the kubernetes documentation:
```
 policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget instead. 
```
Your kubernetes is too high which does not contain `PodDisruptionBudget` of version `policy/v1beta1`.  You can manually disable pdb use the following config:
```
pulsar:
  bookkeeper:
    pdb:
      usePolicy: false
  zookeeper:
    pdb:
      usePolicy: false
  proxy:
    pdb:
      usePolicy: false
  broker:
    pdb:
      usePolicy: false
```

**Comment by Christopher-06** [comment]
Thank you very much for your answer @LoveEachDay 
I have now added your lines to values.yaml and it works perfectly! Big thank you
